readme.md

```diff
@@ -1,49 +1,74 @@
 # Construct
 
+Construct helps you to quickly setup your computer with the tools we have found helpful here at Segment.
 
-Install construct
+Construct will install the following tools:  
 
-  git clone git@github.com:pivotal-cloudops/construct.git
+  atom
+  commandline-tools
+  homebrew
+  homebrew-casks
+  iterm
+  google-chrome
+  recordit
+  slack
+  vagrant
+  virtualbox
 
-Then run it
 
-    ./construct
+To install Construct:
 
-## construct Your Machine
+  1.  To open your terminal press `command` + `space` and type `terminal`.  Then press enter.
 
+  2.  Install Git.  In your terminal type:
+    `git` + enter
+    If it is not already installed it will prompt you to install.
 
-Construct supports blacklisting units for special installation needs: units in `_blacklist` will be skipped
+  3.  In your terminal type:
+    `git clone https://github.com/TeresaNesteby/segment-construct.git`
+
+  3.  In your terminal type:
+    `cd segment-construct`
+
+  4.  Next, type:
+    `./construct`
+
+  5.  Done. Watch as the downloading begins!
+
```

Comment:
You may wish to add something explaining that they may (will) be prompted for this password this is normal.  The password is the password for your computer.